static/app/views/insights/http/views/httpLandingPage.tsx

```diff
@@ -51,15 +53,26 @@ export function HTTPLandingPage() {
   const query = useLocationQuery({
     fields: {
       'span.domain': decodeScalar,
+      [SpanMetricsField.USER_GEO_SUBREGION]: decodeList,
     },
   });
 
+  const ADDITONAL_FILTERS = {
+    ...(query[SpanMetricsField.USER_GEO_SUBREGION].length > 0
+      ? {
+          [SpanMetricsField.USER_GEO_SUBREGION]: `[${query[SpanMetricsField.USER_GEO_SUBREGION].join(',')}]`,
+        }
+      : {}),
+  };
```

Comment:
this block of code is pretty tough to read despite it doing something fairly simple, can we just use an if statement instead? i.e.


```suggestion

  const ADDITIONAL_FILTERS = {};
  
  if (query[SpanMetricsField.USER_GEO_SUBREGION].length > 0) {
    ADDITIONAL_FILTERS[SpanMetricsField.USER_GEO_SUBREGION] = `[${query[SpanMetricsField.USER_GEO_SUBREGION].join(',')}]`
  }
```

Reply:
@0Calories yeah that looks a lot cleaner! I'll keep your suggestion

I think another way to clean it up would be to use a variables, I feel like all the extra spacing and newlines from the formatter makes it harder to read.  For example
```
  const {USER_GEO_SUBREGION} = SpanMetricsField;
  const subregions = query[SpanMetricsField.USER_GEO_SUBREGION]
  
  const ADDITONAL_FILTERS = {
    ...(subregions.length > 0 ? { [USER_GEO_SUBREGION]: `[${subregions.join(',')}]` }: {}),
  };
```